wp_api/src/jetpack/client.rs

```diff
@@ -40,9 +40,9 @@ struct UniffiJetpackApiClient {
 #[uniffi::export]
 impl UniffiJetpackApiClient {
     #[uniffi::constructor]
-    fn new(site_url: Arc<ParsedUrl>, delegate: WpApiClientDelegate) -> Self {
+    fn new(api_root_url: Arc<ParsedUrl>, delegate: WpApiClientDelegate) -> Self {
```

Comment:
Rest of the constructors use `api_root_url`, so I've updated this for consistency.